ABO/BankAccountParser.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JakubZapletal\Component\BankStatement\ABO;
+
+class BankAccountParser implements BankAccountParserInterface
+{
+    const KB = '0100';
```

Comment:
I think this constant is useless if not used anywhere else

Reply:
As it is used only once now, I'l remove it. For constants - comment below :)